File: PHP/Program/Anggota.php
<?php
class Anggota {
    private $id;
    private $nama;
    private $namaBidang;
    private $namaPartai;
    private $foto_profil;

    function __construct($id = '', $nama = "", $namaBidang = "", $namaPartai = "", $foto_profil = "") {
        $this->id = $id;
        $this->nama = $nama;
        $this->namaBidang = $namaBidang;
        $this->namaPartai = $namaPartai;
        $this->foto_profil = $foto_profil;
    }

    //getter and setter
    function get_id() {
        return $this->id;
    }

    function set_id($id) {
        $this->id = $id;
    }

    function get_nama() {
        return $this->nama;
    }

    function set_nama($nama) {
        $this->nama = $nama;
    }

    function get_namaBidang() {
        return $this->namaBidang;
    }

    function set_namaBidang($namaBidang) {
        $this->namaBidang = $namaBidang;
    }

    function get_namaPartai() {
        return $this->namaPartai;
    }

    function set_namaPartai($namaPartai) {
        $this->namaPartai = $namaPartai;
    }
    function get_fotoProfil() {
        return $this->foto_profil;
    }

    function set_fotoProfil($foto_profil) {
        $this->foto_profil = $foto_profil;
    }
}

?>

File: Python/Program/Anggota.py
class Anggota:
    __id = ''
    __nama = ""
    __namaBidang = ""
    __namaPartai = ""
    
    def __init__(self, id = '', nama = "", namaBidang = "", namaPartai = ""):
        self.__id = id
        self.__nama = nama
        self.__namaBidang = namaBidang
        self.__namaPartai = namaPartai

    #getter and setter
    
    def get_id(self):
        return self.__id
    
    def set_id(self, id):
        self.__id = id
    
    def get_nama(self):
        return self.__nama
        
    def set_nama(self, nama):
        self.__nama = nama
        
    def get_namaBidang(self):
        return self.__namaBidang
        
    def set_namaBidang(self, namaBidang):
        self.__namaBidang = namaBidang
        
    def get_namaPartai(self):
        return self.__namaPartai
        
    def set_namaPartai(self, namaPartai):
        self.__namaPartai= namaPartai
        
    